client/src/components/DisplayTableUtils.js

```diff
@@ -0,0 +1,21 @@
+import "./DisplayTableUtils.scss";
+
+import { WriteToClipboard } from "./WriteToClipboard.js";
+import { IconCopy } from "../icons/icons.js";
+
+export class DisplayTableUtils extends React.Component {
```

Comment:
For organization's sake, I'd recommend defining this component in `DisplayTable.js` and not exporting it (doesn't seem like it would have much reason to be used in any other context). Alternatively, create a `/DisplayTable` sub-dir to the `/components` dir and group this and display table there.

Reply:
yeah I think I'd rather make the sub directory to separate it because I expect this component to get larger and I'd rather make `DisplayTable.js` cleaner